tests/cmdline/commands/test_setup.py

```diff
@@ -206,11 +206,15 @@ def test_setup_profile_uuid(self):
 
         profile_name = 'profile-copy'
         user_email = 'some@email.com'
+        user_first_name = 'John'
+        user_last_name = 'Smith'
+        user_institution = 'ECMA'
         profile_uuid = str(uuid.uuid4)
```

Comment:
I did this unrelated fix in my last PR, but it  is better to do it here since you're touching this code

```suggestion
         profile_uuid = str(uuid.uuid4())
```

Also the last line of this file should be an assert I think